.babelrc

```diff
@@ -1,21 +1,31 @@
 {
-  "presets": ["babel-preset-expo"],
+  "presets": [
+    "babel-preset-react-native-stage-0/decorator-support"
+  ],
   "plugins": [
     [
       "module-resolver",
       {
+        "root": "./../../",
```

Comment:
This relative path feels soooo weird, seemingly pointing outside the tree! Is this because it's actually called from a deeper place in the tree?

Not sure how it works though, I removed the line completely and the build+tests+app runs fine.

Reply:
Have you tried running `yarn start --reset-cache` after removing the line?

Reply:
Tried just now, works fine without the `root` line.

Reply:
Ok tested again, and it works now - I removed the root in 01a2454